src/util/fs.js

```diff
@@ -253,6 +253,11 @@ async function buildActionsForCopy(
       }
     }
 
+    if (destStat && destStat.isSymbolicLink()) {
+      await unlink(dest);
+      destStat = null;
```

Comment:
if we don't, the `destStat` (ie. the stat object that contains the infos regarding `dest`) will be wrong, since we've just deleted it.